### DISCUSSION TITLE
What is the `extras` Pipfile?

### DISCUSSION THREAD

**Discussion by stokito** [pypa/pipenv#6236] [Q&A] [answered]
I'm new to Python and trying to run a project.

I have a dependency in the Pipfile:

     somelib = { version = "1.0.0", extras = ["another-lib"] }

As far I understood this should install the somelib and the another-lib with a same version.
Is my assumption correct?
I didn't found anything in the https://pipenv.pypa.io/en/stable/ and anywhere else. It would be really great if anyone can document this.

**Comment by matteius** [comment, marked as answer]
@stokito some packages define extra dependencies named by specific groups, and that is the purpose of the extras list in the package dictionary.
